internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1672,7 +1672,7 @@ func clusterStopped(clusterInstall *hiveext.AgentClusterInstall, status string)
 		condStatus = corev1.ConditionTrue
 		reason = hiveext.ClusterStoppedCanceledReason
 		msg = hiveext.ClusterStoppedCanceledMsg
-	case models.ClusterStatusInstalled:
+	case models.ClusterStatusInstalled, models.ClusterStatusAddingHosts:
 		condStatus = corev1.ConditionTrue
 		reason = hiveext.ClusterStoppedCompletedReason
 		msg = hiveext.ClusterStoppedCompletedMsg
```

Comment:
Perhaps a day-2 cluster should have a different "Stopped" message and reason? Something along the lines of "A day-2 adding hosts cluster is not actively being installed"

Reply:
The status on the ACI is referring to day1 installation, and that installation completed.  The ClusterStatusAddingHosts state is an unfortunate implementation detail in the backend.  If and when additional workers are added to this cluster, the cluster state in the DB won't change, and the ACI Conditions won't either - those are tracked via Agent CRs.

Reply:
>The status on the ACI is referring to day1 installation, and that installation completed.

So are you saying that users are not aware at all that ACI has anything to do with the day-2 flow? Should they only treat the `ClusterDeployment` as their "day-2" object to which they bind future agents?

Reply:
we can look at installation stopped as "the masters are installed"
IMO stopped is a confusing condition to begin with, but if already have it then we can treat it as the installation  of day1

Reply:
> So are you saying that as far as users are concerned ACI has anything to do with the day-2 flow? Should they only treat the ClusterDeployment as their "day-2" object to which they bind future agents and the ACI is 100% completely meaningless in that day 2 context?

The CD and ACI Specs have information needed for adding hosts, but the CD and ACI Status refer only to day1.  Status for day2 is only in the Agents.